en/profile.ftl

```diff
@@ -117,9 +117,13 @@ profile-promo-email-blocking-option-none = None
 profile-promo-email-blocking-description-all-2 = { -brand-name-relay } is blocking all emails sent to this mask.
 profile-promo-email-blocking-description-promotionals = { -brand-name-relay } will attempt to block promotional emails while still forwarding emails like receipts and shipping information.
 profile-promo-email-blocking-description-none-2 = { -brand-name-relay } is not blocking any emails for this mask.
+profile-promo-email-blocking-description-none-3 = All emails sent to this mask will be forwarded to your real address.
 profile-promo-email-blocking-label-promotionals = Block promotions
+profile-promo-email-blocking-label-promotionals-2 = Blocking promo emails
```

Comment:
I don't understand this.

Please add a comment that this is a about blocking promotional emails.

Reply:
added